### PR TITLE
Support remapping all severity levels

### PR DESCRIPTION
**Pull request by dawedawe**
- mark all CLI args as unique
- print messages when using FSC args, too
- support remapping of all severity levels
- support passing of multiple --project flags
- support passing of multiple --analyzers-path flags

### CODE REVIEW

File: src/FSharp.Analyzers.Cli/Program.fs [hunk 1]
```diff
@@ -11,14 +11,18 @@ open Microsoft.CodeAnalysis.Sarif.Writers
...
 type Arguments =
-    | Project of string list
-    | Analyzers_Path of string list
-    | Fail_On_Warnings of string list
-    | Ignore_Files of string list
-    | Exclude_Analyzer of string list
-    | Report of string
-    | FSC_Args of string
-    | Verbose
+    | [<Unique>] Project of string list
```
Comment (baronfel) [inline src/FSharp.Analyzers.Cli/Program.fs:14]: does `Unique` require only a single instance of each of these flags? It's often nicer/easier for users to specify the same flag multiple times, aggregating all of the values of each time the flag is provided. 

```console
> app --project myproject.fsproj --project anotherproject.fsproj
```

is the same as

```console
> app --project myproject.fsproj,anotherproject.fsproj
```

**Review by nojaf — COMMENTED** [review]
This looks good. But I'm a little lost by the `--project` changes.

File: src/FSharp.Analyzers.Cli/Program.fs [hunk 2]
```diff
@@ -398,7 +398,11 @@ let main argv =
     let ignoreFiles = ignoreFiles |> List.map Glob
 
     let analyzersPaths =
-        results.GetResult(<@ Analyzers_Path @>, [ "packages/Analyzers" ])
+        results.GetResults(<@ Analyzers_Path @>)
```
[the remark this answers is not in the document]
Comment (dawedawe) [reply src/FSharp.Analyzers.Cli/Program.fs:401]: Before the last one won:
`--analyzers-path x1 x2 --analyzers-path y1 y2 --analyzers-path z1 z2`
--> we use only z1 z2
With this PR, we use: x1 x2 y1 y2 z1 z2
The same for `--project`

See https://fsprojects.github.io/Argu/reference/argu-parseresults-1.html#GetResult

[the remark this answers is not in the document]
Comment (nojaf) [reply src/FSharp.Analyzers.Cli/Program.fs:401]: Gotcha!

**Review by nojaf — APPROVED** [review]
Looks good

**Pull request merged**